.travis.yml

```diff
@@ -3,8 +3,18 @@ os:
   - linux
   - osx
 node_js:
+  - "8.11.3"
   - "8"
   - "10"
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.12.3
+  - export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"
+cache:
+  yarn: true
+install:
+  - yarn install
+  - cd embark-ui && yarn install && cd ..
+  - git status && test -z "$(git status --porcelain)"
```

Comment:
If after `yarn install` has been run there are unstaged changes or untracked files, etc., CI will stop and report failure. The idea is to catch situations where a commit should have been made for `yarn.lock` but wasn't.